golem/task/acl.py

```diff
@@ -31,6 +57,14 @@ def disallow(self, node_id: str, timeout_seconds: int, persist: bool) \
             -> None:
         pass
 
+    @abc.abstractmethod
+    def allow(self, node_id: str, persist: bool) -> None:
+        pass
```

Comment:
raise NotImplemented
```suggestion
        raise NotImplementedError
```
see https://github.com/golemfactory/golem/pull/4455
and https://docs.python.org/3/library/exceptions.html#NotImplementedError

Reply:
I'd agree with @tworec here

Reply:
I'll add it in #4588